certified-connectors/Fliplet/apiDefinition.swagger.json

```diff
@@ -0,0 +1,408 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Fliplet REST APIs",
```

Comment:
@squallstar Please remove the word "API". please match all names in all files (readme, swagger, and folder name). I would suggest you use "Fliplet". Please see [here ](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#give-your-connector-a-title)and [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#connector-info).